cargo-shuttle/src/lib.rs

```diff
@@ -1416,9 +1435,7 @@ impl Shuttle {
                 service_name.as_str(),
                 false,
                 false,
-                // Set beta to false to avoid breaking local run with beta changes.
-                // TODO: make local run compatible with --beta.
-                false
```

Comment:
Is the comment no longer accurate?

Reply:
Updating resource tables (if we want tables at all) for beta is still TODO, so whatever happens here will change.